doc/widgets/topicmodelling.rst

```diff
@@ -53,21 +57,28 @@ Description
 Example
 -------
 
-In the first example, we present a simple use of the **Topic Modelling** widget. First we load *book-excerpts.tab* data set and use :doc:`Preprocess Text <preprocesstext>` to tokenize by words only. Then we connect **Preprocess Text** to **Topic Modelling**, where we use a simple *Latent Semantic Indexing* to find 10 topics in the text.
+In the first example, we present a simple use of the **Topic Modelling** widget. First we load *grimm-tales-selected.tab* data set and use :doc:`Preprocess Text <preprocesstext>` to tokenize by words only and remove stopwords. Then we connect **Preprocess Text** to **Topic Modelling**, where we use a simple *Latent Semantic Indexing* to find 10 topics in the text.
 
 .. figure:: images/Topic-Modelling-Example1.png
 
-We then select the first topic and display the most frequent words in the topic in :doc:`Word Cloud <wordcloud>`. We also connected **Preprocess Text** to **Word Cloud** in order to be able to output selected documents. Now we can select a specific word in the word cloud, say *polly*. It will be colored red and also highlighted in the word list on the left.
+LSI provides both positive and negative weights per topic. A positive weight means the word is highly representative of a topic, while a negative weight means the word is highly unrepresentative of a topic (the less it occurs in a text, the more likely the topic). Positive words are colored green and negative words are colored red.
+
+We then select the first topic and display the most frequent words in the topic in :doc:`Word Cloud <wordcloud>`. We also connected **Preprocess Text** to **Word Cloud** in order to be able to output selected documents. Now we can select a specific word in the word cloud, say *little*. It will be colored red and also highlighted in the word list on the left.
+
+Now we can observe all the documents containing the word *little* in :doc:`Corpus Viewer <corpusviewer>`.
+
+In the second example we will observe topic distribution across words and documents. Connect **Topic Modelling** to **Heat Map**. Ensure the link is set to *All Topics** - *Data*. **Topic Modelling** will output a matrix of topic weights by words from text (more precisely, tokens).
```

Comment:
I would rephrase `observe topic distribution across words and documents`. Perhaps `In the second example, we will look at the correlation between words (documents) and topics.`?